packages/steps/src/cli/cli.ts

```diff
@@ -23,7 +23,9 @@ export class CliContextProvider implements ExternalBuildContextProvider {
     public readonly projectSourceDirectory: string,
     public readonly projectTargetDirectory: string,
     public readonly defaultWorkingDirectory: string,
-    public readonly buildLogsDirectory: string
+    public readonly buildLogsDirectory: string,
+    public readonly buildDirectory: string,
+    public readonly projectRootDirectory: string
```

Comment:
Oh no, not more directories! haha

What is the reason to add them? Could we use one of the existing directories instead?

If not, can you add comments to those directories explaining what they are? The way we manage directories now is a mess and I find it hard to keep track of what is what.